.travis.yml

```diff
@@ -98,11 +98,20 @@ jobs:
     - &02-php-unit
       stage: 02 unit test
       php: 7.2
+      addons:
+        apt:
+          sources:
+            - sourceline: "deb http://archive.ubuntu.com/ubuntu bionic multiverse"
+            - sourceline: "deb http://archive.ubuntu.com/ubuntu bionic-updates multiverse"
+          packages:
+            - imagemagick
+            - php-imagick
```

Comment:
how is this different than the `pecl install imagick`?

Reply:
It looks like pecl install is all thats needed.  I removed the other lines.